Tmain/list-kind-full.d/stdout-expected.txt

```diff
@@ -13,4 +13,7 @@ t	typedef	typedefs	on	referenceOnly:FALSE	nRoles:0
 u	union	union names	on	referenceOnly:FALSE	nRoles:0
 v	variable	variable definitions	on	referenceOnly:FALSE	nRoles:0
 x	externvar	external and forward variable declarations	off	referenceOnly:FALSE	nRoles:0
-L	label	goto label	off	referenceOnly:FALSE	nRoles:0
+L	label	goto labels	off	referenceOnly:FALSE	nRoles:0
+U	using	using statements	off	referenceOnly:FALSE	nRoles:0
+z	parameter	function parameters inside function definitions	off	referenceOnly:FALSE	nRoles:0
+A	NONE	make anonymous structure names depend on file name	off	referenceOnly:FALSE	nRoles:0
```

Comment:
Having a method to pass additional options to a single parser would be nice.

As for resetting the id... it's even a worse, because the anonymous structure names collide even within a single ctags run (and are still different in different runs). In principle the ctags user can look at the "file" extra field and tweak the structure names by himself but that makes using ctags harder (and almost impossible to do by hand).

On the other side, I could remove the A switch and just make the structure names always depend on the file name. But then it would be a non backward-compatible change...

In the long run having a method for passing other kind of parameters to a single parser is a win.